### PR TITLE
WINDUP-3701 Doc - Analysis summary extract

### PR DESCRIPTION
**Pull request by sbeskin-redhat**
Resolves: https://issues.redhat.com/browse/WINDUP-3701
MTR 1.1.0
Preview:
https://deploy-preview-694--windup-documentation.netlify.app/docs/cli-guide-mtr/master/index.html#cli-args_cli-guide
https://deploy-preview-694--windup-documentation.netlify.app/docs/web-console-guide-mtr/master/index.html#web-create-project_web-console-guide

### CODE REVIEW

**Comment by PhilipCattanach** [comment]
PR reviewed - all good

**Review by stillalearner — APPROVED** [review]
lgtm

**Comment by anarnold97** [comment]
Apart from the pedantic commas... LGTM

File: docs/topics/web-create-project.adoc [hunk 1]
```diff
@@ -64,6 +64,7 @@ Registering the server path ensures that the {ProductShortName} server always us
 * *Mavenize group ID*
 * *Ignore path*: Enter a path for files to exclude from analysis.
 * *Export CSV*: Exports the report data as a CSV file.
+* *Export Summary*: Generate an `analysisSummary.json` export file in the output directory. The file contains analysis summary information for each application analyzed including the number of incidents and story points by category and all of the technology tags associated with the analyzed applications.
```
[the remark this answers is not in the document]
Comment (sbeskin-redhat) [reply docs/topics/web-create-project.adoc]: Yes, there probably should be. Added.

**Review by HagayVider1 — COMMENTED** [review]
Let the reader catch his breath. Include comas between clauses.

File: docs/topics/web-create-project.adoc [hunk 2]
```diff
@@ -64,6 +64,7 @@ Registering the server path ensures that the {ProductShortName} server always us
 * *Mavenize group ID*
 * *Ignore path*: Enter a path for files to exclude from analysis.
 * *Export CSV*: Exports the report data as a CSV file.
+* *Export Summary*: Generate an `analysisSummary.json` export file in the output directory. The file contains analysis summary information for each application analyzed, including the number of incidents and story points by category and all of the technology tags associated with the analyzed applications.
```
[the remark this answers is not in the document]
Comment (sbeskin-redhat) [reply docs/topics/web-create-project.adoc:67]: @HagayVider1 
These are not clauses - why do they need commas? But I can replace the first "and" with a comma and then put an Oxford comma before the second "and".

**Comment by sbeskin-redhat** [comment]
@HagayVider1 
"number of incidents" and "story points" belong together. I've added a comma before the second "and" for clarity.

**Pull request merged**